### PR TITLE
WIP: decompile sub_8106B34

### PR DESCRIPTION
**Pull request by SethBarberee**
Pretty new to this so guidance would be appreciated. I've got most of the assembly to match except any instruction that is supposed to update flags (MOVS/ADDS) doesn't match with mine. I don't know where to go so posting my code here for people to review. 

Preview of compiled code:
```
sub_8106B34:
	push	{lr}
	ldr	r0, .L802
	ldrh	r1, [r0, #0x2e]
	mov	r0, #0x8
	and	r0, r0, r1
	cmp	r0, #0
	beq	.L801	@cond_branch
	ldr	r0, .L802+0x4
	ldr	r0, [r0]
	add	r0, r0, #0x5a
	ldrh	r0, [r0]
	mov	r1, #0x0
	mov	r2, #0x7d
	mov	r3, #0xa
	bl	PlayCry2
.L801:
	pop	{r0}
	bx	r0
.L803:
	.align	2, 0
.L802:
	.word	gMain
	.word	gUnknown_203ACF0
```

### CODE REVIEW

**Comment by PikalaxALT** [comment]
For guidance, please come to the discord. Link in the README.

**Comment by SethBarberee** [comment]
Yes, I've joined the discord - > sesese9#1678

I guess I should ask first in the discord before I post the PR. Felt like it would be easier to point to a PR and ask what I'm doing wrong. Guess I'll just post commits and ask next time.

File: src/pokedex_screen.c [hunk 1]
```diff
@@ -2117,3 +2117,9 @@ bool8 sub_81052D0(u8 a0)
     }
     return FALSE;
 }
+
+void sub_8106B34(void){
+    if(gMain.newKeys & 0x8){
```
Comment (PikalaxALT) [inline src/pokedex_screen.c:2122]: Style guide: space between `if/for/while/do` keywords and opening parenthesis
Repository knowledge: `gMain.newKeys & 0x8` --> `JOY_NEW(START_BUTTON)`

Comment (SethBarberee) [reply src/pokedex_screen.c]: Fixed

**Comment by PikalaxALT** [comment]
Closing for inactivity.

**Pull request closed without merging**